### PR TITLE
Adapt to deprecation of spurt in upstream Mojolicious

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by kalikiana** [comment]
Hrm. How do I find out who can do this?

image

**Comment by kraih** [comment]
The Mojolicious dependency needs to be 9.34 for this change.

**Comment by okurz** [comment]
> The Mojolicious dependency needs to be 9.34 for this change.

done that

File: Changes
Comment (kraih) [inline Changes:3]: The maintainer writes the Changes file, and 2.15 is not even released yet.

File: lib/Mojolicious/Plugin/AssetPack.pm
Comment (kraih) [inline lib/Mojolicious/Plugin/AssetPack.pm:9]: Wrong version.

**Review by kraih — DISMISSED** [review]

File: Changes
Comment (okurz) [reply Changes]: done

**Comment by nicomen** [comment]
perltify fails: 

```diff
$ perltidy --pro=.../.perltidyrc -b -bext='/' **/*.p[lm] **/*.t && git diff --exit-code
diff --git a/lib/Mojolicious/Plugin/AssetPack.pm b/lib/Mojolicious/Plugin/AssetPack.pm
index 4453106..18bbe92 100644
--- a/lib/Mojolicious/Plugin/AssetPack.pm
+++ b/lib/Mojolicious/Plugin/AssetPack.pm
@@ -192,7 +192,7 @@ sub _render_tags {
 
   return Mojo::ByteStream->new(
     join "\n",
-    map    { $_->tag_for->($_, $c, \%args, @attrs) }
+    map { $_->tag_for->($_, $c, \%args, @attrs) }
       grep { !$_->isa('Mojolicious::Plugin::AssetPack::Asset::Null') } @$assets
   );
 }
```

**Comment by kraih** [comment]
We need to fix the CI setup before any patches can be merged.

**Comment by nicomen** [comment]
> We need to fix the CI setup before any patches can be merged.

Yes, the CI setup is fixed in #153 - perhaps you could take a look and see if the fixes seem reasonable.

**Comment by kraih** [comment]
Rebase on main.

**Comment by okurz** [comment]
rebased.

**Pull request merged**
